packages/table/src/tableBody.tsx

```diff
@@ -32,6 +32,12 @@ export interface ITableBodyProps extends ISelectableProps, IRowIndices, IColumnI
     grid: Grid;
 
     /**
+     * If true, all `Cell` render their loading state except for those who have
```

Comment:
happens a few times in this PR: "all singular" sounds wrong. i usually write "all `Cell`s" to follow english grammar rules.